archetypes/database-mp/src/main/resources/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -1,8 +1,8 @@
 
 # Datasource properties
 javax.sql.DataSource.test.dataSourceClassName=org.h2.jdbcx.JdbcDataSource
-#javax.sql.DataSource.test.dataSource.url=jdbc:h2:tcp://localhost:1521/test;IFEXISTS=FALSE
 javax.sql.DataSource.test.dataSource.url=jdbc:h2:mem:test;DB_CLOSE_DELAY=-1
+#javax.sql.DataSource.test.dataSource.url=jdbc:h2:tcp://localhost:1521/test;IFEXISTS=FALSE
```

Comment:
ifexists is not needed here, sufficient on server side

Reply:
That's odd, had issues without it. Will re-try.